processing/src/main/java/org/apache/carbondata/processing/util/CarbonDataProcessorUtil.java

```diff
@@ -375,6 +375,25 @@ public static boolean isHeaderValid(String tableName, String[] csvHeader,
     return type.toArray(new DataType[type.size()]);
   }
 
+  /**
+   * get visible no dictionary dimensions as per data field order
+   *
+   * @param dataFields
+   * @return
+   */
+  public static DataType[] getNoDictDataTypesAsDataFieldOrder(DataField[] dataFields) {
+    List<DataType> type = new ArrayList<>();
+    for (DataField dataField : dataFields) {
```

Comment:
can use steam

Reply:
This is base code. This file has same function with for loops.